pkg/registry/manifests.go

```diff
@@ -147,3 +160,35 @@ func downloadManifests(manifestURLs []string, destPath string) ([]string, error)
 
 	return manifestPaths, nil
 }
+
+func CopyManifests(src string, dest string) ([]string, error) {
```

Comment:
Is all of this replaceable by `fileio.CopyFiles()`?

Reply:
The benefit of this function is that it checks for `.yaml` and `.yml` + appends a tag to local files to prevent duplicate names. The previously mentioned `local-manifest` and `downloaded-manifest` solution ends up being moot cause all of the files end up in `...rke2/servers/manifests`.